### ISSUE TITLE
iAtom implementations of adapton don't return new value

### ISSUE THREAD

**Issue by den1k** [roman01la/uix#57]
**Describe the bug**
Calls to either `clojure.core/reset!` or `clojure.core/swap!` are supposed to return the updated value of the atom-like object. It looks like the implementations in xframe do not:

https://github.com/roman01la/uix/blob/master/core/src/xframe/core/adapton.clj#L63-L75

**Include minimal repro**
```clojure
(reset! (atom {}) {:foo :bar})
=> {:foo :bar}

(reset! (xframe.core.adapton/aref {}) {:foo :bar})
; => nil <-------------- wrong
```

**Expected behavior**
Same return value as with `clojure.core/atom`

(I've only tested this in CLJ.)

**Comment by roman01la** [comment]
Oooops, thanks! PR with tests is welcomed!

**Comment by roman01la** [comment]
fixed in #58
